crates/maybenot-ffi/Makefile

```diff
@@ -0,0 +1,30 @@
+CARGO ?= cargo
+TARGET ?=
+PROFILE ?= release
+DESTINATION ?= .
+
+CARGOFLAGS += --locked
+TARGET_DIR := ../../target/$(TARGET)/$(PROFILE)
```

Comment:
Given the choice between these options, which one do you think would suck the least? x)
- The makefile does not respect the CARGO_TARGET_DIR env
- `make clean` will clean your global cargo cache
- `make clean` will not clean any cargo cache
- `make clean` only runs `cargo clean` if CARGO_TARGET_ENV is a relative path

Reply:
As discussed in the office (want to get it written down): We *must* respect `CARGO_TARGET_DIR` since we set it in our build environments. How we handle cleaning is not super important. It's fine to wipe everything since it's not destructive.

Reply:
:+1: 
Went with option 2